submissions/Harnytskyi/html_css_popup/css/style.css

```diff
@@ -0,0 +1,115 @@
+.menu{
```

Comment:
Between selector and a bracket should be a space. Could you please format this file by any auto formatting?